services/compute/virtual-machines/taxonomy.md

```diff
@@ -0,0 +1,47 @@
+# Virtual Machines Taxonomy
+
+This _service-level taxonomy_ documents the minimal set of features
+that should be present for a service to be considered portable for
+use in financial services ecosystems.
+
+## Taxonomy
+
+| Taxonomy ID | Feature | Description |
+| ----------- | ------- | ----------- |
+|	CCC-030101	|	Instance Types	|	Offers a range of instance types with different specifications for CPU, GPU, memory, storage optimisation and networking capacity.	|
```

Comment:
I have second thoughts how validation tests could be written to test this. Instead should we have like
General Computing
High Performance Computing
Storage Optimized Computing
Low Latency Network Computing
Would like to know what others think of this.

Reply:
Agree that there's potentially a lot to validate by leaving this as one category. A couple of these features are definitely quite hefty, I'll work on splitting this out.

Reply:
Different providers have different views on what constitutes HPC or HiMem etc.
It might need to be some more fuzzy requirement!

Reply:
Should we wait and see what the Taxonomy WG thinks about this?